pages/common/flyctl.md

```diff
@@ -0,0 +1,36 @@
+# flyctl
+
+> Command-line tool for flyctl.io.
+> More information: <https://github.com/superfly/flyctl>.
+
+- Sign into a Fly account:
+
+`flyctl auth login`
+
+- Launch an application from a specific Dockerfile (the default path is the current working directory):
+
+`flyctl launch --dockerfile {{path/to/dockerfile}}`
+
+- Open the browser to the current deployed application:
+
+`flyctl open`
+
+- Deploy the Fly applications from a specific Dockerfile:
+
+`flyctl deploy --dockerfile {{path/to/dockerfile}}`
+
+- Open the Fly Web UI for the current application in a web browser:
+
+`flyctl dashboard`
+
+- List all applications in your Fly.io account:
```

Comment:
```suggestion
- List all applications in the logged-in Fly account:
```
We try to avoid personal pronouns.
Also, you used `Fly` in the first example, so we should keep it consistent. You could also change both to `Fly.io`